environments/prod/hearings-hmcts-net.yml

```diff
@@ -350,6 +350,9 @@ cname:
   - name: _22CCCAD51C677A40E9922BAC36584E24.sip.dev.self-test
     ttl: 300
     record: 9434C251F0A49AC87C2A51251159AE70.CC04A93DB03503D0FAF61D02B7EFF8CE.603a60c30fc626443b46.comodoca.com.
+  - name: vh-user-api-dev
```

Comment:
is this an internal or external API?

Reply:
this is internal, so I assume the this is the wrong place ?

Reply:
yes it is, we haven't done load balancing for internal yet, for now you will just have to point the DNS record at one of the ingress controllers, the repo is hmcts/azure-private-dns

Reply:
i'll delete this PR